src/DynamoCoreWpf/Utilities/CrashUtilities.cs

```diff
@@ -26,28 +29,61 @@ internal static string GithubNewIssueUrlFromCrashContent(object crashContent)
             return baseUri.ToString();
         }
 
+        /// <summary>
+        /// Converts packages information into markdown format for use in Github issue body
+        /// </summary>
+        /// <param name="packageLoader">Package loader</param>
+        /// <returns>A markdown format string to use in issue body.</returns>
+        internal static string PackagesToMakrdown(PackageLoader packageLoader)
+        {
+            if (packageLoader != null)
+            {
+                //List of the names of all the loaded packages
+                var packagesNames = packageLoader.LocalPackages.Select(o => o.Name).ToList();
+                //Package's issue section in markdown format
+                string markdownText;
+                if (packagesNames.Any())
+                {
+                    markdownText = "### Loaded Packages" + Environment.NewLine;
+                    foreach (var name in packagesNames)
+                    {
+                        markdownText += "- " + name + Environment.NewLine;
+                    }
+                }
+                else
+                {
+                    markdownText = "No loaded packages were found.";
```

Comment:
So I thought about if we need to convert these into resource strings but think it may the customer but not us. What do you guys think @DynamoDS/dynamo